fides/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: fides
-version: 0.17.0
+version: 0.17.1
 appVersion: "2.64.0"
```

While we're at it, we might as well 

```suggestion
appVersion: "2.67.0"
```